starter_code/routes/index.js

```diff
@@ -1,9 +1,44 @@
 const express = require('express');
 const router  = express.Router();
+const Movie = require('../models/Movie');
+const app_name = require('../package.json').name;
+const debug = require('debug')(`${app_name}:indexRouter`);
 
 /* GET home page */
 router.get('/', (req, res, next) => {
-  res.render('index');
+  debug(process.env.DEBUG);
+  let data = {
+    name: "home"
+  };
+  res.render('index',{data});
 });
 
+/* GET movies page */
+router.get('/movies', (req, res, next) => {
+  debug(process.env.DEBUG);
```

Comment:
**Clean code** delete the console.logs and debugs, unless necessary :wink: Try to only keep those who give you information about errors, but avoid to keep those who bring you the data when you send a PR (pull request)